lib/tapioca/generators/base.rb

```diff
@@ -10,13 +10,18 @@ class Base
       extend T::Sig
       extend T::Helpers
 
+      class FileWriter < Thor
+        include Thor::Actions
+      end
```

Comment:
Are we okay with FileWriter having access to all Thor::Actions instead of just `create_file`?

Reply:
I would say "yes", if only because we can update other helpers to take advantage of the built in `Thor::Actions`.

i.e. `create_directory`, etc.

Reply:
👍
Yeah I don't love the name of the class since it can do more. But it's a nitpick.